<ui version="4.0" >
 <class>ReportDlg</class>
 <widget class="QDialog" name="ReportDlg" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>194</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>291</width>
    <height>194</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>291</width>
    <height>194</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>291</width>
    <height>194</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Generate Report</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" native="1" name="widget" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>158</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title" >
    <string>Generate Report</string>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>69</x>
      <y>20</y>
      <width>122</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Report Period Start&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
    </property>
   </widget>
   <widget class="QDateEdit" name="startDate" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::WheelFocus</enum>
    </property>
   </widget>
   <widget class="QTimeEdit" name="startTime" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::TabFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>66</x>
      <y>88</y>
      <width>122</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Report Period End&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDate" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::TabFocus</enum>
    </property>
   </widget>
   <widget class="QTimeEdit" name="endTime" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::TabFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="genButton" >
   <property name="geometry" >
    <rect>
     <x>53</x>
     <y>157</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Generate Report</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>165</x>
     <y>157</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
